proto/substrait/expression.proto

```diff
@@ -18,7 +18,8 @@ message Expression {
     SwitchExpression switch_expression = 7;
     SingularOrList singular_or_list = 8;
     MultiOrList multi_or_list = 9;
-    Enum enum = 10;
+    Enum enum
+    = 10;
```

Comment:
This seems ... off.

Reply:
Whoops, I left clang-format on write. I do think it'd be beneficial to check in a clang-format configuration to substrait, though